### PR TITLE
CARDS #41 & #43: created boilerplate

### PR DESCRIPTION
**Pull request by CarsonDSaunders**
just added a backend directory and npm init'd it.

### CODE REVIEW

**Comment by CarsonDSaunders** [comment]
@rogersmykenzie @bheadwhite 

Hey y'all. Could y'all please review this PR? It's a hefty one.

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Switch>
```
Comment (rogersmykenzie) [inline frontend/src/App.tsx:11]: Put this in routes file

File: frontend/src/components/EndpointTesting.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react'
```
Comment (rogersmykenzie) [inline frontend/src/components/EndpointTesting.tsx:1]: is this just a test file? if so do we need it in PR

File: backend/test/MyRoom_test.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import assert from "assert";
+import { ColyseusTestServer, boot } from "@colyseus/testing";
+
+// import your "arena.config.ts" file here.
+import appConfig from "../src/arena.config";
+import { MyRoomState } from "../src/rooms/schema/MyRoomState";
+
+describe("testing your Colyseus app", () => {
```
Comment (rogersmykenzie) [inline backend/test/MyRoom_test.ts:8]: This is strange test, as it looks like it is testing just Colyseus behavior and not custom functionality, which should be priority of tests.

Comment (CarsonDSaunders) [reply backend/test/MyRoom_test.ts:8]: Good point, I went ahead and removed that entire directory (it came packaged with the template)

File: frontend/src/App.tsx (hunk 1 above)
Comment (CarsonDSaunders) [reply frontend/src/App.tsx:11]: Done

File: frontend/src/components/EndpointTesting.tsx (hunk 2 above)
Comment (CarsonDSaunders) [reply frontend/src/components/EndpointTesting.tsx:1]: Removed from PR

**Pull request merged**